proto/elys/leveragelp/tx.proto

```diff
@@ -94,33 +98,39 @@ message MsgUpdateStopLoss {
 message MsgUpdateStopLossResponse {}
 
 message MsgAddCollateral {
-  string creator = 1;
-  uint64 id      = 2;
+  string creator    = 1;
+  uint64 id         = 2;
   string collateral = 3 [(gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int", (gogoproto.nullable) = false];
 }
 
 message MsgAddCollateralResponse {}
 
 message AddPool {
-  uint64 amm_pool_id = 1;
-  bool enabled = 2;
-  bool closed = 3;
-  string leverage_max = 4 [
-    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
-    (gogoproto.nullable) = false
-  ];
+  uint64 amm_pool_id  = 1;
+  bool   enabled      = 2;
+  bool   closed       = 3;
+  string leverage_max = 4 [(gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec", (gogoproto.nullable) = false];
 }
 
 message MsgAddPool {
-  string authority = 1;
-  AddPool pool = 2[(gogoproto.nullable) = false];
+  string  authority = 1;
+  AddPool pool      = 2 [(gogoproto.nullable) = false];
 }
 
 message MsgAddPoolResponse {}
 
 message MsgRemovePool {
   string authority = 1;
-  uint64 id      = 2;
+  uint64 id        = 2;
 }
 
-message MsgRemovePoolResponse {}
\ No newline at end of file
+message MsgRemovePoolResponse {}
+
+message MsgClosePositions {
+  string creator   = 1;
+  repeated PositionRequest liquidate = 2;
```

Comment:
@amityadav0 `PositionRequest` is a type defined under `query.proto` can you move it within the `types.proto` file as it will be used for both query and tx messages from now on, thanks

Reply:
ok